Automattic/vip-go-mu-plugins
# 2FA: Don't check wpcom_vip_is_two_factor_forced() for machine user

Don't check the wpcom_vip_is_two_factor_forced() if we already know the current user is a machine user.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Checkout this PR
1. Confirm caps are limited without 2FA
1. Confirm caps are not limited with 2FA
